### PR TITLE
[AMDGPU] clang-tidy: no else after return etc. NFC.

### PR DESCRIPTION
**Pull request by jayfoad**

### CODE REVIEW

File: llvm/lib/Target/AMDGPU/R600InstrInfo.cpp [hunk 1]
```diff
@@ -739,38 +740,36 @@ unsigned R600InstrInfo::insertBranch(MachineBasicBlock &MBB,
...
     BuildMI(&MBB, DL, get(R600::JUMP_COND))
-            .addMBB(TBB)
-            .addReg(R600::PREDICATE_BIT, RegState::Kill);
-    BuildMI(&MBB, DL, get(R600::JUMP)).addMBB(FBB);
+        .addMBB(TBB)
+        .addReg(R600::PREDICATE_BIT, RegState::Kill);
     MachineBasicBlock::iterator CfAlu = FindLastAluClause(MBB);
     if (CfAlu == MBB.end())
-      return 2;
+      return 1;
```
Comment (rampitec) [inline llvm/lib/Target/AMDGPU/R600InstrInfo.cpp:754]: This is a functional change? And the same below.

Comment (slinder1) [reply llvm/lib/Target/AMDGPU/R600InstrInfo.cpp:754]: I think it is NFC, but git's diff is matching up the wrong lines. This is the "inner" version that is nested under the "then" part of `if (!FBB)`, but it is being compared against the "else" version from the original.

Comment (rampitec) [reply llvm/lib/Target/AMDGPU/R600InstrInfo.cpp:754]: I see, thanks!

**Review by rampitec — APPROVED** [review]
LGTM

File: llvm/lib/Target/AMDGPU/R600InstrInfo.cpp (hunk 1 above)
Comment (jayfoad) [reply llvm/lib/Target/AMDGPU/R600InstrInfo.cpp:754]: Yeah, you can get a more readable diff for that part if you select "Hide whitespace" under the gear wheel icon at the top of the whole diff.

**Pull request merged**